src/views/Main/Link/Link.js

```diff
@@ -0,0 +1,316 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import Select from 'react-select';
+import Promise from 'bluebird';
+import { EntityDataModelApi } from 'loom-data';
+import Page from '../../../components/page/Page';
+import buttonStyles from '../../../core/styles/buttons.css';
+import styles from './styles.module.css';
+
+export class Link extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
```

Comment:
Since the start of January, we decided that new features will be built using Redux. Most of what you have here in `state` should go in the Redux store.